plugins/patch-github-integration-plugin.yml

```diff
@@ -0,0 +1,86 @@
+apiVersion: spinnaker.armory.io/v1alpha2
+kind: SpinnakerService
+metadata:
+  name: spinnaker
+spec:
+  spinnakerConfig:
+    # Configs in the spinnaker profile get applied to all services
+    spinnaker:
+      spinnaker:
+        extensibility:
+          repositories:
+            repository:
+              enabled: true
+              url: https://raw.githubusercontent.com/armory-plugins/pluginRepository/master/repositories.json
```

Comment:
@christosarvanitis Does this patch need the spec.spinnakerConfig.profiles.deck.settings-local.js ?

```yaml
spec:
  spinnakerConfig:
    profiles:
      spinnaker:
        github:
          plugin:
            accounts: []
        spinnaker:
          extensibility:
            repositories:
              repository:
                enabled: true
                url: https://raw.githubusercontent.com/armory-plugins/pluginRepository/master/repositories.json
      deck:
        settings-local.js:
```

Or does the user add that someplace else for Armory CD?

Reply:
I would recommend that we  document what to do here for someone that hasn't already set the Deck plugin settings, as they may not be aware of the trickiness with this file.

Reply:
Followed the same pattern as the rest of the patches in this repo.